comfy/option.py

```diff
@@ -5,37 +5,50 @@
 
 class Option(BaseOption):
 
-    def __get__(self, section, type=None):
-        # type: (Section, Optional[type]) -> str
-        config_parser = section.config_parser
-        return config_parser.get(section.name, self.name)
+    def unserialize(self, value):
+        return value
```

Comment:
Please add mypy-compatible type annotations! In this case:

    from typing import Text
    # ...
    # type: (Text) -> Text